src/components/shared/drag-upload/drag-upload.tsx

```diff
@@ -52,7 +52,7 @@ export const DragUpload = forwardRef<IDragUploadRef, IDragUploadProps>((props, r
 				aria-hidden="true"
 				tabIndex={-1}
 				capture={capture}
-				accept={accept?.map((type) => `.${type}`).join(",")}
+				accept={accept?.map((type) => `image/${type}`).join(",")}
```

Comment:
The reason we need that is related to an issue on Android devices (see more [here](https://stackoverflow.com/questions/77876374/html-input-type-file-not-working-to-pull-up-camera-for-pixel-android-14-comb)): if both accept and capture properties exist at the same time, then accept needs to be full image MIME type otherwise even the capture attribute is set correctly, the device will open the gallery instead of the camera.

Any suggestions to resolve this issue?

Reply:
I see, since this component can now handle images only, may I suggest some refactoring:
- Move `drag-upload` to within `image-input/` folder
- Update the `accept` type from `string[]` to `TImageUploadAcceptedFileType[]`
- Leave a comment above the `accept` attribute to explain the bug you mentioned

Reply:
We had a discussion with the team yesterday about this. Would it be better if the onus is on the parent component returning `drag-upload` to provide the correctly formatted `accept` value instead of mapping it in `drag-upload`? That way `drag-upload` can still support other file types, and we can provide the full MIME type in the accept prop when returning it in `image-input`.

Reply:
hello it's fine to narrow `drag-upload` to support images only because it is unlikely to be used by other component in the future. this is because there is a `FileUpload` component from design system to handle any kinds of file uploads.

in the future, `drag-upload` along with some components in `image-upload` will be deprecated when we adapt it to be based on the `file-upload` field.